inputs/select.vue

```diff
@@ -89,37 +94,58 @@
     props: ['name', 'data', 'schema', 'args'],
     data() {
       return {
-        options: [],
+        listOptions: [],
         isDisabled: false
       };
     },
     mounted() {
       if(this.args.list){
         this.fetchListItems().then( listItems => {
-          this.setOptions(listItems);
-        });
-      } else {
-        this.setOptions();
+          this.listOptions = listItems;
+        }); // todo: ".catch"?
       }
     },
     computed: {
-      safeData() {
-        if (_.isString(this.data)) {
-          return _.find(this.options, (option) => option.value === this.data);
-        } else if (_.isObject(this.data) && !_.isEmpty(this.data)) {
-          return _.reduce(this.options, (safeArray, option) => {
-            if (this.data[option.value] === true) {
-              return safeArray.concat([option]);
-            } else {
-              return safeArray;
-            }
-          }, []);
+      keys () {
+        return {
+          label: 'name', // backwards compatibility I guess?
+          value: 'value',
+          ...this.args.keys
+        }
+      },
+      NULL_OPTION () {
+        return {
+          [this.keys.label]: 'None',
+          [this.keys.value]: null,
+        }
+      }, 
```

Comment:
moved this into a computed property because it's construction depends on `args.keys`